plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -579,17 +547,57 @@ def finalize(self):
                                            self.context.checksum)
 
 
-class PublishRpmAndDrpmStepIncremental(platform_steps.UnitPublishStep):
+def create_legacy_metadata_dict(unit):
```

Comment:
We should attach this as a method on one of the models. It makes sense to do that for OO design reasons. This seems to be used by rpm, srpm, drpm, and erratum so if we attach it onto `plugins.pulp_rpm.plugins.db.models.Package` it will be available in All the Right Places™